### PR TITLE
Dpkt

### PR DESCRIPTION
**Pull request by kylekeppler**
Adding receipe for dpkt https://github.com/kbandla/dpkt

@conda-forge/help-python

### CODE REVIEW

**Comment by kylekeppler** [comment]
Looks like the appveyor build failure is not related to this PR.  Can it be re-ran?

**Comment by beckermr** [comment]
Going!

**Comment by kylekeppler** [comment]
Does this PR look good? Anyone have comments? Is there a way I can download the build artifact and test it?

File: recipes/dpkt/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    dpkt is a simple packet creation / parsing, with definitions for the basic TCP/IP protocols
+    and is implemented in pure python
+  doc_url: https://dpkt.readthedocs.io
+  dev_url: http://github.com/kbandla/dpkt
+
+extra:
+  recipe-maintainers:
+    - brifordwylie
+    - obormot
+    - kylekeppler
```
Comment (beckermr) [inline recipes/dpkt/meta.yaml]: @brifordwylie and @obormot need to confirm they are OK being maintainers

Comment (kylekeppler) [reply recipes/dpkt/meta.yaml]: @brifordwylie, @obormot, you cool with being listed here?

Comment (obormot) [reply recipes/dpkt/meta.yaml]: I'm an active maintainer of dpkt itself, but I'm not using conda personally so listing me as a *recipe* maintainer here would be a stretch..  I'd rather not.  Thanks for checking!

Comment (beckermr) [reply recipes/dpkt/meta.yaml]: Great. @kylekeppler can you please remove @obormot from the list?

**Comment by kylekeppler** [comment]
Will do.

**Comment by kylekeppler** [comment]
@beckermr, any idea on these build failures? Seem unrelated to my PR.

**Comment by beckermr** [comment]
Yeah. Rebase this PR on master and try again. We have been struggling with some upstream conda upgrades etc.

**Comment by beckermr** [comment]
Hmmmmm. This PR looks wrong. Maybe checkout a new branch and copy paste there?

**Comment by kylekeppler** [comment]
I think I probably messed up the rebase. Using Sourcetree it looks like it merged master into my dpkt branch because it had already been pushed so all those commits are showing. Crap.

**Comment by beckermr** [comment]
Yeah this is hard! Just copy paste to a new PR. Close this one and make a reference to the new one in the comments. Path of least resistance!

**Comment by beckermr** [comment]
None of the history in staged recipes really matters FWIW. Once the recipe is merged all dev goes to the feedstock repo anyways.

**Pull request closed without merging**